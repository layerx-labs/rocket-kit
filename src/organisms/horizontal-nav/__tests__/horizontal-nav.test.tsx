import React from 'react';
import { render } from '@testing-library/react';
import { HorizontalNav, Icon } from '../../..';

const menu = [
  {
    icon: 'overview',
    label: 'Overview',
    url: '#0',
    active: true,
  },
  {
    icon: 'timeline',
    label: 'Timeline',
    url: '#0',
    active: false,
  },
  {
    icon: 'rules',
    label: 'Rules',
    url: '#0',
    active: false,
  },
  {
    icon: 'resources',
    label: 'Resources',
    url: '#0',
    active: false,
  },
  {
    icon: 'projects',
    label: 'Projects',
    url: '#0',
    active: false,
  },
  {
    icon: 'users',
    label: 'Innovators',
    url: '#0',
    active: false,
  },
  {
    icon: 'backer',
    label: 'Backers',
    url: '#0',
    active: false,
  },
  {
    icon: 'faqs',
    label: 'Faqs',
    url: '#0',
    active: false,
  },
  {
    icon: 'notifications',
    label: 'Updates',
    url: '#0',
    active: false,
  },
  {
    icon: 'kai',
    label: 'Transactions',
    url: '#0',
    active: false,
  },
  {
    icon: 'trophy',
    label: 'Results',
    url: '#0',
    active: false,
  },
];

describe('Horizontal Nav', () => {
  it('renders', () => {
    const { asFragment } = render(
      <HorizontalNav
        items={menu.map((item, index) => (
          <li key={index} className={item.active ? 'active' : ''}>
            <a href={item.url}>
              {item.icon && <Icon icon={item.icon} />}
              {item.label}
            </a>
          </li>
        ))}
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
