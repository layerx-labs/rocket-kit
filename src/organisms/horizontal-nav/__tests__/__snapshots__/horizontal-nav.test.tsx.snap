// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Horizontal Nav renders 1`] = `
<DocumentFragment>
  .c2 {
  --button: hsl(186,62%,59%);
  --txt: hsl(0,0%,100%);
  --hover: #2ba8b6;
  --button: hsl(0,0%,16%);
  --hover: #000;
  border-width: 0;
  border-style: solid;
  border-color: var(--button);
  border-radius: 0;
  background-color: transparent;
  width: 2.25rem;
  height: 2.25rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0;
  text-transform: uppercase;
  white-space: nowrap;
  -webkit-transition-duration: 0.3s;
  transition-duration: 0.3s;
  cursor: pointer;
}

.c2:hover {
  border-color: var(--hover);
  background-color: transparent;
}

.c2:hover span {
  color: var(--hover);
}

.c2:hover svg {
  fill: var(--hover);
}

.c2:disabled {
  cursor: inherit;
  opacity: 0.5;
}

.c2:disabled:hover {
  pointer-events: none;
}

.c2 > *:not(:last-child) {
  margin-left: 0;
  margin-right: 5px;
}

.c2 .spinner {
  border-top-color: var(--button);
}

.c2 span {
  position: relative;
  font-size: 0.75rem;
  font-weight: 700;
  color: var(--button);
  pointer-events: none;
  -webkit-transition-duration: 0.3s;
  transition-duration: 0.3s;
  -webkit-order: 2;
  -ms-flex-order: 2;
  order: 2;
}

.c2 svg {
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
  width: auto;
  min-width: 1.25rem;
  height: 1.25rem;
  fill: var(--button);
  -webkit-transition: 0.3s;
  transition: 0.3s;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 ul {
  margin: 0;
  width: 0;
  max-width: -webkit-max-content;
  max-width: -moz-max-content;
  max-width: max-content;
  padding: 0;
  overflow: hidden;
  -webkit-animation: showAnimation 1s forwards;
  animation: showAnimation 1s forwards;
  -webkit-animation-delay: 0.5s;
  animation-delay: 0.5s;
}

.c0 ul.menu {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c0 ul.menu li:not(:last-child) {
  margin-right: 0.625rem;
}

.c0 ul li {
  position: relative;
  list-style: none;
}

.c0 ul li a {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0.625rem;
  color: hsl(0,0%,16%);
  text-transform: capitalize;
  -webkit-text-decoration: none;
  text-decoration: none;
  white-space: nowrap;
  -webkit-transition-duration: 0.3s;
  transition-duration: 0.3s;
}

.c0 ul li a svg {
  margin-right: 0.3125rem;
  width: auto;
  height: 1.5rem;
  min-width: 1.5rem;
  fill: #e0e0e0;
  -webkit-transition-duration: 0.3s;
  transition-duration: 0.3s;
}

.c0 ul li a:hover {
  color: hsl(186,62%,59%);
}

.c0 ul li a:hover svg {
  fill: hsl(186,62%,59%);
}

.c0 ul li.active {
  font-weight: 700;
}

.c0 ul li.active a {
  pointer-events: none;
}

.c0 ul li.active a svg {
  fill: hsl(186,62%,59%);
}

.c1 {
  position: relative;
  display: none;
}

.c1.hide {
  display: initial;
}

.c1 ul {
  position: absolute;
  right: 0;
  border: 1px solid #e0e0e0;
  border-radius: 4px;
  background-color: hsl(0,0%,100%);
  margin-top: 5px;
  padding: 0;
  -moz-box-shadow: 0 0 15px 0 rgba(40,40,40,0.2);
  -webkit-box-shadow: 0 0 15px 0 rgba(40,40,40,0.2);
  box-shadow: 0 0 15px 0 rgba(40,40,40,0.2);
  z-index: 1;
  overflow-x: hidden;
  overflow-y: auto;
  -webkit-scrollbar-width: none;
  -moz-scrollbar-width: none;
  -ms-scrollbar-width: none;
  scrollbar-width: none;
  -webkit-overflow-scrolling: touch;
  -ms-overflow-style: -ms-autohiding-scrollbar;
}

.c1 ul.more {
  display: none;
}

.c1 ul.is-open {
  display: inherit;
}

.c1 ul li {
  list-style: none;
  -webkit-transition-duration: 0.3s;
  transition-duration: 0.3s;
}

.c1 ul li:hover {
  background-color: #e0e0e0;
  cursor: pointer;
}

.c1 ul li:hover a {
  color: hsl(0,0%,16%);
}

.c1 ul li:hover a svg {
  fill: hsl(0,0%,16%);
}

.c1 ul li:first-child {
  border-radius: 4px 4px 0 0;
}

.c1 ul li:last-child {
  border-radius: 0 0 4px 4px;
}

.c1 ul li a {
  width: 100%;
  height: 100%;
  padding: 15px;
}

@media (min-width:1280px) {
  .c2 {
    min-width: 2.25rem;
  }
}

<div
    class="c0"
  >
    <ul
      class="menu"
    >
      <li
        class="active"
      >
        <a
          href="#0"
        >
          <svg
            class="icon"
            viewBox="0 0 32 32"
          >
            <path
              d="M17.333 22.667h-10.667c-0.733 0-1.333 0.6-1.333 1.333s0.6 1.333 1.333 1.333h10.667c0.733 0 1.333-0.6 1.333-1.333s-0.6-1.333-1.333-1.333zM25.333 12h-18.667c-0.733 0-1.333 0.6-1.333 1.333s0.6 1.333 1.333 1.333h18.667c0.733 0 1.333-0.6 1.333-1.333s-0.6-1.333-1.333-1.333zM6.667 20h18.667c0.733 0 1.333-0.6 1.333-1.333s-0.6-1.333-1.333-1.333h-18.667c-0.733 0-1.333 0.6-1.333 1.333s0.6 1.333 1.333 1.333zM5.333 8c0 0.733 0.6 1.333 1.333 1.333h18.667c0.733 0 1.333-0.6 1.333-1.333s-0.6-1.333-1.333-1.333h-18.667c-0.733 0-1.333 0.6-1.333 1.333z"
            />
          </svg>
          Overview
        </a>
      </li>
    </ul>
    <div
      class="c1 hide"
    >
      <button
        class="c2 button"
        color="dark"
        value=""
      >
        <svg
          class="icon"
          viewBox="0 0 32 32"
        >
          <path
            d="M16 10.667c1.473 0 2.667-1.194 2.667-2.667s-1.194-2.667-2.667-2.667c-1.473 0-2.666 1.194-2.667 2.666v0c0.004 1.471 1.196 2.662 2.666 2.667h0zM16 13.333c-1.473 0-2.667 1.194-2.667 2.667s1.194 2.667 2.667 2.667c1.473 0 2.667-1.194 2.667-2.667v0c-0.005-1.471-1.196-2.662-2.666-2.667h-0zM16 21.333c-1.473 0-2.667 1.194-2.667 2.667s1.194 2.667 2.667 2.667c1.473 0 2.667-1.194 2.667-2.667v0c-0.005-1.471-1.196-2.662-2.666-2.667h-0z"
          />
        </svg>
      </button>
      <ul
        class="more"
      >
        <li
          class=""
        >
          <a
            href="#0"
          >
            <svg
              class="icon"
              viewBox="0 0 32 32"
            >
              <path
                d="M15.987 2.667c-7.36 0-13.32 5.973-13.32 13.333s5.96 13.333 13.32 13.333c7.373 0 13.347-5.973 13.347-13.333s-5.973-13.333-13.347-13.333zM16 26.667c-5.893 0-10.667-4.773-10.667-10.667s4.773-10.667 10.667-10.667 10.667 4.773 10.667 10.667-4.773 10.667-10.667 10.667zM15.707 9.333h-0.080c-0.533 0-0.96 0.427-0.96 0.96v6.293c0 0.467 0.24 0.907 0.653 1.147l5.533 3.32c0.453 0.267 1.040 0.133 1.307-0.32 0.28-0.453 0.133-1.053-0.333-1.32l-5.16-3.067v-6.053c0-0.533-0.427-0.96-0.96-0.96z"
              />
            </svg>
            Timeline
          </a>
        </li>
        <li
          class=""
        >
          <a
            href="#0"
          >
            <svg
              class="icon"
              viewBox="0 0 32 32"
            >
              <path
                d="M28 8.4c0-1.053-0.627-2.013-1.587-2.44l-9.333-4.147c-0.693-0.307-1.48-0.307-2.16 0l-9.333 4.147c-0.96 0.427-1.587 1.387-1.587 2.44v6.267c0 7.4 5.12 14.32 12 16 3.067-0.747 5.773-2.533 7.84-4.947l-4.16-4.16c-2.587 1.72-6.107 1.427-8.387-0.853-2.6-2.6-2.6-6.827 0-9.427s6.827-2.6 9.427 0c2.28 2.28 2.56 5.8 0.853 8.387l3.867 3.867c1.613-2.613 2.56-5.693 2.56-8.867v-6.267z M20 16c0 2.209-1.791 4-4 4s-4-1.791-4-4c0-2.209 1.791-4 4-4s4 1.791 4 4z"
              />
            </svg>
            Rules
          </a>
        </li>
        <li
          class=""
        >
          <a
            href="#0"
          >
            <svg
              class="icon"
              viewBox="0 0 32 32"
            >
              <path
                d="M22 9v14.107c0 2.787-2.040 5.267-4.813 5.533-3.187 0.307-5.853-2.187-5.853-5.307v-16.48c0-1.747 1.253-3.333 2.987-3.507 2-0.2 3.68 1.36 3.68 3.32v14c0 0.733-0.6 1.333-1.333 1.333s-1.333-0.6-1.333-1.333v-11.667c0-0.547-0.453-1-1-1s-1 0.453-1 1v11.48c0 1.747 1.253 3.333 2.987 3.507 2 0.2 3.68-1.36 3.68-3.32v-13.773c0-2.787-2.040-5.267-4.813-5.533-3.173-0.307-5.853 2.187-5.853 5.307v16.36c0 3.827 2.8 7.253 6.613 7.613 4.387 0.4 8.053-3.013 8.053-7.307v-14.333c0-0.547-0.453-1-1-1s-1 0.453-1 1z"
              />
            </svg>
            Resources
          </a>
        </li>
        <li
          class=""
        >
          <a
            href="#0"
          >
            <svg
              class="icon"
              viewBox="0 0 32 32"
            >
              <path
                d="M26.333 5.613c-0.241-0.24-0.573-0.388-0.94-0.388s-0.699 0.148-0.94 0.388l-0.52 0.52c-0.238 0.239-0.385 0.569-0.385 0.933s0.147 0.694 0.385 0.933l0.013 0.013c0.241 0.24 0.573 0.388 0.94 0.388s0.699-0.148 0.94-0.388l0.52-0.52c0.236-0.24 0.381-0.57 0.381-0.933 0-0.37-0.151-0.705-0.394-0.947l-0-0zM16.013 1.333h-0.013c-0.002 0-0.003 0-0.005 0-0.726 0-1.315 0.589-1.315 1.315 0 0.002 0 0.004 0 0.006v-0 1.28c0 0.002 0 0.003 0 0.005 0 0.726 0.589 1.315 1.315 1.315 0.002 0 0.004 0 0.006 0h0.013c0.002 0 0.003 0 0.005 0 0.726 0 1.315-0.589 1.315-1.315 0-0.002 0-0.004 0-0.006v0-1.28c0-0.002 0-0.003 0-0.005 0-0.726-0.589-1.315-1.315-1.315-0.002 0-0.004 0-0.006 0h0zM27.987 17.267h1.36c0.002 0 0.003 0 0.005 0 0.726 0 1.315-0.589 1.315-1.315 0-0.002 0-0.004 0-0.006v0-0.013c0-0.002 0-0.003 0-0.005 0-0.726-0.589-1.315-1.315-1.315-0.002 0-0.004 0-0.006 0h-1.346c-0.002 0-0.003 0-0.005 0-0.726 0-1.315 0.589-1.315 1.315 0 0.002 0 0.004 0 0.006v-0 0.013c-0 0.008-0 0.016-0 0.025 0 0.715 0.58 1.295 1.295 1.295 0.004 0 0.009-0 0.013-0h-0.001zM12 22.853v5.080c0 0.736 0.597 1.333 1.333 1.333v0h5.333c0.736 0 1.333-0.597 1.333-1.333v0-5.080c2.408-1.409 4-3.983 4-6.928 0-4.418-3.582-8-8-8s-8 3.582-8 8c0 2.946 1.592 5.519 3.962 6.908l0.038 0.020zM5.333 15.947v-0.013c0-0.002 0-0.003 0-0.005 0-0.726-0.589-1.315-1.315-1.315-0.002 0-0.004 0-0.006 0h-1.346c-0.002 0-0.003 0-0.005 0-0.726 0-1.315 0.589-1.315 1.315 0 0.002 0 0.004 0 0.006v-0 0.013c0 0.002 0 0.003 0 0.005 0 0.726 0.589 1.315 1.315 1.315 0.002 0 0.004 0 0.006 0h1.346c0.002 0 0.003 0 0.005 0 0.726 0 1.315-0.589 1.315-1.315 0-0.002 0-0.004 0-0.006v0zM8.080 6.12l-0.52-0.52c-0.232-0.199-0.535-0.321-0.867-0.321-0.736 0-1.333 0.597-1.333 1.333 0 0.332 0.121 0.636 0.322 0.869l-0.001-0.002 0.52 0.52c0.232 0.199 0.535 0.321 0.867 0.321 0.736 0 1.333-0.597 1.333-1.333 0-0.332-0.121-0.636-0.322-0.869l0.001 0.002z"
              />
            </svg>
            Projects
          </a>
        </li>
        <li
          class=""
        >
          <a
            href="#0"
          >
            <svg
              class="icon"
              viewBox="0 0 32 32"
            >
              <path
                d="M21.333 14.667c2.213 0 3.987-1.787 3.987-4s-1.773-4-3.987-4-4 1.787-4 4 1.787 4 4 4zM10.667 14.667c2.213 0 3.987-1.787 3.987-4s-1.773-4-3.987-4-4 1.787-4 4 1.787 4 4 4zM10.667 17.333c-3.107 0-9.333 1.56-9.333 4.667v2c0 0.733 0.6 1.333 1.333 1.333h16c0.733 0 1.333-0.6 1.333-1.333v-2c0-3.107-6.227-4.667-9.333-4.667zM21.333 17.333c-0.387 0-0.827 0.027-1.293 0.067 0.027 0.013 0.040 0.040 0.053 0.053 1.52 1.107 2.573 2.587 2.573 4.547v2c0 0.467-0.093 0.92-0.24 1.333h6.907c0.733 0 1.333-0.6 1.333-1.333v-2c0-3.107-6.227-4.667-9.333-4.667z"
              />
            </svg>
            Innovators
          </a>
        </li>
        <li
          class=""
        >
          <a
            href="#0"
          >
            <svg
              class="icon"
              viewBox="0 0 32 32"
            >
              <path
                d="M15.987 2.667c-7.358 0.007-13.32 5.974-13.32 13.333 0 7.364 5.97 13.333 13.333 13.333s13.333-5.97 13.333-13.333v0c0-0.002 0-0.004 0-0.007 0-7.36-5.967-13.327-13.327-13.327-0.007 0-0.014 0-0.021 0h0.001zM16 26.667c-5.891 0-10.667-4.776-10.667-10.667s4.776-10.667 10.667-10.667c5.891 0 10.667 4.776 10.667 10.667v0c0 0.001 0 0.002 0 0.003 0 5.889-4.774 10.664-10.664 10.664-0.001 0-0.002 0-0.003 0h0zM16 23.333c2.694-0.003 5.047-1.462 6.314-3.632l0.019-0.035c0.054-0.094 0.085-0.208 0.085-0.328 0-0.371-0.301-0.672-0.672-0.672-0 0-0 0-0 0h-11.493c-0.371 0.003-0.67 0.304-0.67 0.675 0 0.119 0.031 0.231 0.085 0.329l-0.002-0.003c1.287 2.205 3.639 3.663 6.333 3.667h0z M18.132 9.592c0.3 0.004 0.542 0.246 0.546 0.545v2.389l3.094-2.799c0.039-0.031 0.084-0.059 0.131-0.080l0.004-0.002c0.067-0.032 0.146-0.051 0.229-0.051 0 0 0.001 0 0.001 0h-0c0.152 0.004 0.288 0.066 0.388 0.165l-0-0c0.011 0.010 0.022 0.021 0.032 0.032l0 0c0.079 0.094 0.126 0.216 0.126 0.349 0 0.107-0.030 0.206-0.083 0.29l0.001-0.002c-0.016 0.024-0.033 0.045-0.051 0.064l0-0c-0.021 0.025-0.043 0.046-0.068 0.066l-0.001 0.001-2.359 2.13 0.604 0.6 0.549-0.494c0.097-0.085 0.224-0.137 0.364-0.137 0.16 0 0.305 0.068 0.406 0.178l0 0c0.084 0.091 0.135 0.214 0.135 0.348 0 0.016-0.001 0.032-0.002 0.047l0-0.002c-0.009 0.148-0.077 0.279-0.18 0.37l-0.001 0-0.504 0.454 1.012 1.004 0.021 0.032 0.011 0.011c0.086 0.092 0.139 0.216 0.139 0.352 0 0.008-0 0.016-0.001 0.023l0-0.001c-0.003 0.145-0.064 0.276-0.16 0.37l-0 0c-0.087 0.090-0.207 0.148-0.34 0.157l-0.002 0c-0.013 0.001-0.028 0.002-0.043 0.002-0.114 0-0.22-0.035-0.307-0.094l0.002 0.001-0.019-0.015-0.009-0.008-0.020-0.014c-0.009-0.007-0.017-0.014-0.032-0.028l-1.064-1.060-0.547 0.49c-0.095 0.087-0.223 0.141-0.363 0.141-0.009 0-0.017-0-0.026-0.001l0.001 0c-0.15-0.003-0.284-0.069-0.377-0.173l-0-0.001c-0.085-0.097-0.138-0.225-0.138-0.365 0-0.16 0.068-0.305 0.178-0.406l0-0 0.505-0.455-0.603-0.597-0.637 0.576v1.458c0 0.301-0.244 0.546-0.546 0.546s-0.546-0.244-0.546-0.546v0-5.315c0.004-0.3 0.245-0.541 0.544-0.546h0z M9.812 9.592c0.3 0.004 0.542 0.246 0.546 0.545v2.389l3.094-2.799c0.039-0.031 0.084-0.059 0.131-0.080l0.004-0.002c0.067-0.032 0.146-0.051 0.229-0.051 0 0 0.001 0 0.001 0h-0c0.152 0.004 0.288 0.066 0.388 0.165l-0-0c0.011 0.010 0.022 0.021 0.032 0.032l0 0c0.079 0.094 0.126 0.216 0.126 0.349 0 0.107-0.030 0.206-0.083 0.29l0.001-0.002c-0.016 0.024-0.033 0.045-0.051 0.064l0-0c-0.021 0.025-0.043 0.046-0.068 0.066l-0.001 0.001-2.359 2.13 0.604 0.6 0.549-0.494c0.097-0.085 0.225-0.137 0.364-0.137 0.16 0 0.305 0.068 0.406 0.178l0 0c0.084 0.091 0.135 0.214 0.135 0.348 0 0.016-0.001 0.032-0.002 0.047l0-0.002c-0.009 0.148-0.077 0.279-0.18 0.37l-0.001 0-0.504 0.454 1.012 1.004 0.021 0.032 0.011 0.011c0.086 0.092 0.139 0.216 0.139 0.352 0 0.008-0 0.016-0.001 0.023l0-0.001c-0.003 0.145-0.064 0.276-0.16 0.37l-0 0c-0.088 0.090-0.207 0.148-0.34 0.157l-0.002 0c-0.013 0.001-0.028 0.002-0.043 0.002-0.114 0-0.22-0.035-0.307-0.094l0.002 0.001-0.019-0.015-0.009-0.008-0.020-0.014c-0.009-0.007-0.017-0.014-0.032-0.028l-1.064-1.060-0.547 0.49c-0.095 0.087-0.223 0.141-0.363 0.141-0.009 0-0.017-0-0.026-0.001l0.001 0c-0.15-0.003-0.284-0.069-0.377-0.173l-0-0.001c-0.085-0.097-0.138-0.225-0.138-0.365 0-0.16 0.068-0.305 0.178-0.406l0-0 0.505-0.455-0.603-0.597-0.637 0.576v1.458c0 0.301-0.244 0.546-0.546 0.546s-0.546-0.244-0.546-0.546v0-5.315c0.004-0.3 0.245-0.541 0.544-0.546h0z"
              />
            </svg>
            Backers
          </a>
        </li>
        <li
          class=""
        >
          <a
            href="#0"
          >
            <svg
              class="icon"
              viewBox="0 0 32 32"
            >
              <path
                d="M16 2.667c-7.36 0-13.333 5.973-13.333 13.333s5.973 13.333 13.333 13.333 13.333-5.973 13.333-13.333-5.973-13.333-13.333-13.333zM16 26.667c-5.88 0-10.667-4.787-10.667-10.667s4.787-10.667 10.667-10.667 10.667 4.787 10.667 10.667-4.787 10.667-10.667 10.667zM14.667 21.333h2.667v2.667h-2.667zM16.813 8.053c-2.747-0.4-5.173 1.293-5.907 3.72-0.24 0.773 0.347 1.56 1.16 1.56h0.267c0.547 0 0.987-0.387 1.173-0.893 0.427-1.187 1.693-2 3.067-1.707 1.267 0.267 2.2 1.507 2.093 2.8-0.133 1.787-2.16 2.173-3.267 3.84 0 0.013-0.013 0.013-0.013 0.027-0.013 0.027-0.027 0.040-0.040 0.067-0.12 0.2-0.24 0.427-0.333 0.667-0.013 0.040-0.040 0.067-0.053 0.107-0.013 0.027-0.013 0.053-0.027 0.093-0.16 0.453-0.267 1-0.267 1.667h2.667c0-0.56 0.147-1.027 0.373-1.427 0.027-0.040 0.040-0.080 0.067-0.12 0.107-0.187 0.24-0.36 0.373-0.52 0.013-0.013 0.027-0.040 0.040-0.053 0.133-0.16 0.28-0.307 0.44-0.453 1.28-1.213 3.013-2.2 2.653-4.747-0.32-2.32-2.147-4.28-4.467-4.627z"
              />
            </svg>
            Faqs
          </a>
        </li>
        <li
          class=""
        >
          <a
            href="#0"
          >
            <svg
              class="icon"
              viewBox="0 0 32 32"
            >
              <path
                d="M16 29.333c1.471-0.005 2.662-1.196 2.667-2.666v-0h-5.333c0 0 0 0 0 0.001 0 1.472 1.194 2.666 2.666 2.666 0 0 0 0 0.001 0h-0zM24 21.333v-6.667c0-4.093-2.187-7.52-6-8.427v-0.907c0-1.105-0.895-2-2-2s-2 0.895-2 2v0 0.907c-3.827 0.907-6 4.32-6 8.427v6.667l-1.72 1.72c-0.244 0.242-0.394 0.577-0.394 0.947 0 0.734 0.594 1.33 1.327 1.333h17.56c0.738-0.001 1.336-0.599 1.336-1.337 0-0.368-0.149-0.701-0.389-0.943l0 0z"
              />
            </svg>
            Updates
          </a>
        </li>
        <li
          class=""
        >
          <a
            href="#0"
          >
            <svg
              class="icon"
              viewBox="0 0 32 32"
            >
              <path
                d="M9.451 5.76c0.973 0.012 1.76 0.798 1.771 1.77v7.755l10.043-9.085c0.127-0.102 0.272-0.191 0.427-0.261l0.013-0.005c0.217-0.105 0.473-0.166 0.742-0.166 0.001 0 0.002 0 0.003 0h-0c0.492 0.012 0.934 0.214 1.258 0.536l-0-0 0.002-0.002c0.037 0.033 0.071 0.067 0.103 0.102l0.001 0.001c0.255 0.305 0.41 0.701 0.41 1.134 0 0.346-0.099 0.669-0.271 0.942l0.004-0.007c-0.053 0.076-0.107 0.143-0.166 0.205l0.001-0.001c-0.068 0.080-0.141 0.151-0.221 0.215l-0.003 0.002-7.656 6.915 1.961 1.947 1.782-1.605c0.314-0.277 0.729-0.446 1.183-0.446 0.521 0 0.989 0.222 1.317 0.576l0.001 0.001c0.273 0.296 0.44 0.693 0.44 1.129 0 0.052-0.002 0.103-0.007 0.153l0-0.006c-0.029 0.48-0.248 0.904-0.583 1.201l-0.002 0.002-1.635 1.472 3.283 3.26 0.069 0.1 0.036 0.036c0.279 0.299 0.45 0.701 0.45 1.143 0 0.026-0.001 0.052-0.002 0.077l0-0.004c-0.012 0.472-0.21 0.895-0.524 1.201l-0 0c-0.284 0.291-0.671 0.48-1.102 0.508l-0.005 0c-0.041 0.003-0.089 0.005-0.137 0.005-0.37 0-0.713-0.112-0.998-0.305l0.006 0.004-0.061-0.048-0.028-0.026c-0.032-0.022-0.010-0.007-0.066-0.047-0.028-0.022-0.055-0.047-0.102-0.092l-3.453-3.443-1.782 1.605c-0.309 0.284-0.722 0.458-1.176 0.458-0.028 0-0.057-0.001-0.085-0.002l0.004 0c-0.487-0.008-0.922-0.224-1.222-0.562l-0.002-0.002c-0.277-0.314-0.446-0.729-0.446-1.184 0-0.52 0.222-0.989 0.576-1.316l0.001-0.001 1.64-1.477-1.952-1.946-2.070 1.876v4.73c0 0.003 0 0.006 0 0.010 0 0.974-0.789 1.763-1.763 1.763-0.491 0-0.935-0.201-1.255-0.524l-0-0c-0.324-0.313-0.525-0.752-0.525-1.237 0-0.004 0-0.008 0-0.011v0.001-17.254c0.012-0.973 0.798-1.76 1.77-1.771h0.001z"
              />
            </svg>
            Transactions
          </a>
        </li>
        <li
          class=""
        >
          <a
            href="#0"
          >
            <svg
              class="icon"
              viewBox="0 0 32 32"
            >
              <path
                d="M27.185 5.789c-0.516-0.724-1.344-1.197-2.283-1.224l-0.004-0h-1.139c-0.031-0.112-0.061-0.224-0.096-0.335-0.098-0.321-0.346-0.568-0.66-0.665l-0.007-0.002c-2.109-0.641-4.534-1.009-7.044-1.009s-4.935 0.369-7.222 1.055l0.178-0.046c-0.321 0.098-0.569 0.346-0.665 0.66l-0.002 0.007c-0.033 0.111-0.064 0.223-0.096 0.335h-1.139c-0.943 0.027-1.77 0.5-2.281 1.215l-0.006 0.009c-0.549 0.778-0.877 1.746-0.877 2.79 0 0.506 0.077 0.994 0.22 1.452l-0.009-0.035 0.021 0.071c0.838 3.063 3.117 5.458 6.045 6.441l0.386 0.113c0.8 0.938 1.792 1.687 2.917 2.192l0.052 0.021c-0.175 1.834-0.697 3.511-1.501 5.015l0.035-0.071-0.479 0.62h-2.332c-0 0-0.001 0-0.001 0-0.548 0-0.992 0.444-0.992 0.992 0 0 0 0 0 0v0 3.375c0 0 0 0.001 0 0.001 0 0.548 0.444 0.992 0.992 0.992 0 0 0.001 0 0.001 0h13.5c0 0 0 0 0 0 0.548 0 0.992-0.444 0.992-0.992 0-0 0-0.001 0-0.001v0-3.375c-0.001-0.548-0.444-0.991-0.992-0.992h-2.133l-0.479-0.62c-0.778-1.458-1.303-3.165-1.469-4.975l-0.004-0.053c1.098-0.528 2.025-1.246 2.773-2.122l0.010-0.011 0.321-0.093c2.992-1.001 5.27-3.394 6.094-6.389l0.015-0.066 0.023-0.076c0.134-0.424 0.211-0.912 0.211-1.418 0-1.043-0.327-2.009-0.885-2.802l0.010 0.016zM5.965 9.492l-0.019-0.060c-0.080-0.254-0.126-0.546-0.126-0.848 0-0.594 0.178-1.146 0.482-1.606l-0.007 0.011c0.152-0.242 0.406-0.407 0.7-0.436l0.004-0h0.737c-0.079 0.578-0.125 1.245-0.125 1.923v0.001c0 0.007-0 0.016-0 0.025 0 1.801 0.36 3.518 1.011 5.083l-0.032-0.088c-1.234-1.022-2.152-2.386-2.611-3.947l-0.014-0.057zM13.959 24.4c0.699-1.477 1.207-3.191 1.434-4.993l0.009-0.084c0.181 0.016 0.365 0.028 0.549 0.028 0.263-0.001 0.522-0.018 0.775-0.051l-0.031 0.003c0.236 1.893 0.744 3.614 1.49 5.202l-0.044-0.105zM25.952 9.431l-0.019 0.063c-0.474 1.616-1.391 2.979-2.611 3.989l-0.013 0.011c0.619-1.477 0.979-3.194 0.979-4.995 0-0.009 0-0.017-0-0.026v0.001c-0-0.679-0.046-1.347-0.134-2.001l0.008 0.077h0.737c0.298 0.029 0.552 0.194 0.702 0.432l0.002 0.004c0.295 0.448 0.471 0.997 0.471 1.587 0 0.306-0.047 0.601-0.135 0.879l0.006-0.021z"
              />
            </svg>
            Results
          </a>
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
